src/components/navigation/CTopbar.vue

```diff
@@ -62,6 +62,9 @@
             </span>
           </div>
         </template>
+        <div>
+          <slot name="tour-dropdown" />
```

Comment:
Since this  slot can be used for anything and not just for the tour. rename this to `help-dropdown`.
Similarly how the slot in the avatar dropdown is named `avatar-dropdown`